spec/scrum_lint/interactive_linters/card/missing_context_spec.rb

```diff
@@ -0,0 +1,110 @@
+RSpec.describe ScrumLint::InteractiveLinter::MissingContext, '#call' do
+  let(:checker) { described_class.new }
+  let(:project_card) { fake_trello_card(name: '#Project', url: 'pro/ject') }
+  let(:project_card_2) { fake_trello_card(name: '#Poopject', url: 'poop/ject') }
+  let(:card) { fake_trello_card(name: '#Butts fart more', desc: 'nocontext') }
+
+  before do
+    allow(Launchy).to receive(:open)
+    allow(card).to receive(:desc=)
+  end
+
+  context 'when there is no project card with matching hashtag' do
+    before do
+      # gets must have a mocked return value or the tests will pass every time
+      allow(checker).to receive(:gets).and_return('2')
+    end
+
+    it 'prints list of project cards' do
+      allow(checker).to receive(:puts).exactly(4).times
+      allow(checker).to receive(:print)
+
+      checker.(card, active_project_cards: [project_card, project_card_2])
+
+      expect(checker).to have_received(:print).with('enter project number > ')
+      expect(checker).to have_received(:puts).with('1. #Project')
+      expect(checker).to have_received(:puts).with('2. #Poopject')
+    end
+
+    it 'adds a context link to the description of the chosen card' do
+      checker.(card, active_project_cards: [project_card, project_card_2])
+
+      expect(card).to have_received(:desc=)
+        .with("Context: #{project_card_2.url}\n\n#{card.desc}")
+    end
+
+    it 'skips and prints message when an invalid option is chosen' do
+      allow(checker).to receive(:gets).and_return('doodypoo')
+      allow(checker).to receive(:puts).exactly(4).times
+
+      checker.(card, active_project_cards: [project_card, project_card_2])
+
+      expected_message = 'doodypoo is not a valid option. Skipping card; ' \
```

Comment:
I disagree. `'doodypoo'` should always be considered valid.

Reply:
hmmmmmm. you're right. i should probably add an `unless 'doodypoo'` conditional.